src/app/shared/components/validate-json-or-yaml/validate-json-or-yaml.component.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, Input, OnChanges, SimpleChanges} from '@angular/core';
+import {verifyYAML, verifyJSON} from '@shared/utils/common';
+
+@Component({
+  selector: 'km-validate-json-or-yaml',
+  templateUrl: './validate-json-or-yaml.component.html',
+  styleUrls: ['./validate-json-or-yaml.component.css'],
+})
+export class ValidateJsonOrYamlComponent implements OnChanges {
+  @Input() model: string;
```

Comment:
can we rename this to 
- `config` or 
- `configModel` or 
- `kubeconfig` (if specifically we are passing kubeconfig`
- `data` 
- `manifest` 
to create k8s' objects via manifest?

Reason:
- model -> has multiple meaning (class model etc)
- We know inside component what we are verifying.

Reply:
feedback implemented